### DISCUSSION TITLE
哥，为什么你的新版文件没有Autovisor.exe？

### DISCUSSION THREAD

**Discussion by leo-cheung-itlger** [CXRunfree/Autovisor#18] [Q&A]
哥，为什么你的新版文件没有Autovisor.exe？

**Comment by CXRunfree** [comment]
你不会是下的源代码吧，要在releases或者网盘里下带有版本号的压缩包。也有可能解压出来的exe文件被杀毒软件干掉了

**Comment by leo-cheung-itlger** [comment]
okok!应该是的，因为活不太会用github!!谢谢你在百忙之中回复我的消息




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年5月13日(星期一) 中午11:24
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [CXRunfree/Autovisor] 哥，为什么你的新版文件没有Autovisor.exe？ (Discussion #18)














 
你不会是下的源代码吧，要在releases或者网盘里下带有版本号的压缩包。也有可能解压出来的exe文件被杀毒软件干掉了
